crates/eframe/src/native/glow_integration.rs

```diff
@@ -48,24 +48,24 @@ use super::{
 // ----------------------------------------------------------------------------
 // Types:
 
-pub struct GlowWinitApp {
+pub struct GlowWinitApp<'a> {
```

Comment:
Let's be explicit and call this lifetime `'app` here and everywhere

Reply:
Good idea, done.